packages/core/contracts/common/interfaces/Uniswap.sol

```diff
@@ -8,4 +8,8 @@ pragma solidity ^0.6.0;
 abstract contract Uniswap {
     // Called after every swap showing the new uniswap "price" for this token pair.
     event Sync(uint112 reserve0, uint112 reserve1);
+    // Base currency.
+    address public token0;
+    // Quote currency.
+    address public token1;
```

Comment:
Based on the rest of the code, isn't this the other way around, with `token0` the quote currency and `token1` the quote currency?

Reply:
@nicholaspai and I discussed this and we differed in our interpretation of how to translate the base and quote currencies into the uniswap feed. So @nicholaspai suggested that we just remove that terminology from the PR to make it easier to reason about.

Reply:
I removed all references to "quote" and "base" currency because my usage was ambiguous. I think the standard is that for the currency pair listed as "X-Y", X is the base and Y is the quote.